sources/api/apiserver/src/server/controller.rs

```diff
@@ -374,6 +378,32 @@ where
     Ok(())
 }
 
+/// Dispatches an update command via `thar-be-updates`
+pub(crate) fn dispatch_update_command(args: Vec<String>) -> Result<HttpResponse> {
+    let status = Command::new("/usr/bin/thar-be-updates")
+        .args(args)
+        .status()
+        .context(error::UpdateDispatcher)?;
+    if status.success() {
+        Ok(HttpResponse::NoContent().finish())
+    } else {
+        let exit_status = match status.code() {
+            Some(code) => code,
+            None => status.signal().unwrap_or(1),
+        };
+        let error_type = FromPrimitive::from_i32(exit_status);
+        let error = match error_type {
+            Some(TbuErrorStatus::LockHeld) => error::Error::UpdateLockHeld,
+            Some(TbuErrorStatus::DisallowedCmd) => error::Error::UpdateActionNotAllowed,
+            Some(TbuErrorStatus::UpdateDoesNotExist) => error::Error::UpdateMissing,
+            Some(TbuErrorStatus::NoStagedImage) => error::Error::NoStagedImage,
```

Comment:
nit: can the errors be named the same?  :)  Easier to track between codebases.

Seems like there are actually three names, for example `DisallowCommand` -> `DisallowedCmd` -> `UpdateActionNotAllowed`, so you have to make multiple jumps right now.

Reply:
There are still some mismatches, like `LockHeld` -> `UpdateLockHeld`, `DisallowCommand` -> `DisallowedCommand`, etc. in `dispatch_update_command`.

Reply:
Ah sorry about that.